addons/l10n_nl/data/account_fiscal_position_account_template.xml

```diff
@@ -92,5 +92,30 @@
             <field name="account_dest_id" ref="8022" />
         </record>
 
+        <!-- Installatie / afstandsverkopen -->
+        <record
+            id="fiscal_position_account_installatie_afstand_1"
+            model="account.fiscal.position.account.template">
```

Comment:
tag can be on one line

Reply:
All the other lines are splitted like this too in the current 12.0, see https://github.com/odoo/odoo/blob/12.0/addons/l10n_nl/data/account_fiscal_position_account_template.xml
Would you like me to correct all of them to a one liner?

Reply:
No, let's leave it like this then.  I would like it differently, but then it should be in a different commit/pr.

Reply:
Indeed. Perhaps we can all reformat them in `master` in another PR?

Reply:
@jco-odoo would you like a PR for the reformat of the file in master?